src/themes/typography.js

```diff
@@ -0,0 +1,38 @@
+import tw from "twin.macro";
+
+const typography = {
```

Comment:
We need to update this so that it fits under the `theme` object (as we discussed by the Button component)

Reply:
fixed